www/sejd00/staticka-html-prezentace/css/style.css

```diff
@@ -21,42 +23,64 @@ a {
     
 }
 
-nav {
-    border: solid black 1px;
-}
-
-a:active{
-    color: red;
-}
-
 a:hover {
     color: gray;
+    transform: scale(1.05);
+}
+
+.social-media-logo:hover{
+    transform: scale(1.05);
 }
 
 .site-description{
     color: white;
 }
 
-
 .wrap{
     background-color: rgba(0, 0, 0, 0.3);
-    margin-top: 100px;
     padding-top: 50px;
     padding-bottom: 50px;
-    padding-left: 10%;
-    margin-right: 50%;
+    padding-left: 10%; 
 }
 
 .bg-main{
-    background-image: url("/4IZ268-2018-2019-ZS/www/sejd00/staticka-html-prezentace/img/cover.jpg");
+    background-image: url("../img/cover1.jpg");
     height: 100%; 
     background-position: center;
     background-repeat: no-repeat;
     background-size: cover;
+    border: solid black 1px;
 }
 
 .bg-about{
-    background-image: url("/4IZ268-2018-2019-ZS/www/sejd00/staticka-html-prezentace/img/cover2.jpg");
+    background-image: url("../img/cover2.jpg");
+    height: 100%; 
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    
+}
+
+.bg-contacts{
+    background-image: url("../img/cover3.jpg");
```

Comment:
prosim zkompresovat vsechny obrazky compressjpeg.com compresspng.com

Reply:
Jaká je optimální velikost obrázku? Už jsem tyto obrázky projel kompresí, očividně jsou stále až moc velké.

Reply:
tak aby uz to neslo vic kompresovat, tj kdyz to hodim do komprese tak bych mel dostat zpatky zhruba stejnou velikost, pred chvili jsem to zkousel a dostal jsem az 50% kompresi